src/ui/components/Library/Team/View/NewTestRuns/Overview/RunResults.tsx

```diff
@@ -206,20 +206,29 @@ function PathNodeRenderer({
     <>
       {name && (
         <div
-          className={`cursor-pointer truncate rounded py-2 pr-4 ${styles.libraryRow} ${
-            containsSelectedSpec ? styles.libraryRowSelected : ""
-          }`}
+          className={`flex cursor-pointer justify-between truncate rounded py-2 pr-4 ${
```

Comment:
I think you can fix the right alignment by replacing "pr-4" with "pr-2"

Reply:
The padding is too much.

Reply:
I see what you mean. I'll be addressing Jon's comment above, so it won't be aligned to right anyway. I'll remove right padding.